content/v2.0/reference/api/client-libraries/csharp-cl.md

```diff
@@ -0,0 +1,9 @@
+---
+title: InfluxDB C# client library
+description: Use the C# client library to interact with InfluxDB.
+menu:
+  v2_0_ref:
+    name: C# client library
```

Comment:
This is nested under client libraries in the left nav, so "client library" here is redundant. I'd just just use the language as the menu name across all of the client libraries.